doc/user/content/third-party/debezium.md

```diff
@@ -34,6 +34,10 @@ ALTER TABLE foo
 REPLICA IDENTITY FULL;
 ```
 
+### Open CDC format
+
+You can use Debezium's default CDC envelope or Materialize's Open CDC format, which has been designed to prevent record duplication and sequencing errors. For more information about Open CDC, see [Open CDC](/overview/open-cdc).
```

Comment:
Ah––this makes way more sense to me now. Ty for this! This makes me feel like we also need a section on configuring Debezium to use this format.

Reply:
There's no way to configure Debezium to use this format. You would have to write code by hand to generate it, or wait until we convince other tools to support it.

Reply:
Ah––what do we expect people to do with this information, then?

Reply:
It's possible for someone (especially a particularly sophisticated user) to write code to generate this format.

Anyway... *shrug*. Documenting it is the first step to getting other people in the industry to use it.

Reply:
Do we have any tests that use this format that we could use as an instructive pointer for the sophisticated users?

Reply:
https://github.com/MaterializeInc/materialize/blob/main/test/testdrive/avro-cdcv2.td

Reply:
nice, ty –– @msudberg I think it's crucial that we have a a link to this as an instruction for someone else who wants to implement support for it, so they can at least see what it looks like in practice

Reply:
We can link to it, but those are the same examples we already list on the overview/open-cdc page.

I misunderstood what my notes about people being able to use the Debezium envelope or Open CDC meant, so I think I should just remove this section.